compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -79,6 +81,40 @@ pub(super) fn check_argument_count(
     }
 }
 
+pub(super) fn check_one_argument(
```

Comment:
I thought it could be a good idea to have these helper functions to reduce some code duplication.